### PR TITLE
Changelog for Azure.Core 1.22.0

### PR DESCRIPTION
**Pull request by annelo-msft**

### CODE REVIEW

File: sdk/core/Azure.Core/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.22.0-beta.1 (Unreleased)
+## 1.22.0 (2022-01-11)
 
 ### Features Added
```
Comment (pakrym) [inline sdk/core/Azure.Core/CHANGELOG.md:5]: Can we mention new ResourceIdentifier, ResouceType, Location types?

**Review by pakrym — APPROVED** [review]

File: sdk/core/Azure.Core/CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 - Added `AddPolicies` method to `RequestContext`.  This allows policies to be added to the pipeline when calling protocol methods.
 - Added `IsError` property to `Response`.  This will indicate whether the message's `ResponseClassifier` considers the response to be an error.
 - Added `RequestFailedException` constructor that takes a `Response`.
+- Added types `AzureLocation`, `ResourceIdentifier`, and `ResourceType`.
```
Comment (annelo-msft) [inline sdk/core/Azure.Core/CHANGELOG.md:10]: @m-nash let me know if you'd like more detail here.

**Comment by christothes** [comment]
Suggested addition to changelog:

### Features Added
- Added `HttpPipelineTransportOptions` which contains a `ServerCertificateCustomValidationCallback` property allowing configuration of a `Func<ServerCertificateCustomValidationArgs, bool>` delegate to be called when validating the server side TLS certificate.
- Added a new static overload for `HttpPipelineBuilder.Build` taking an `HttpPipelineTransportOptions` instance to construct the `HttpPipeline` with the default transport configurations and the `HttpPipelineTransportOptions` applied. This `Build` method returns a `DisposableHttpPipeline` which implements `IDisposable`. Note: The `HttpPipelineTransportOptions` will only be applied if a custom `Transport` has not been set in the `ClientOptions`. In the case that transport options were provided but not applied, an event is logged `(PipelineTransportOptionsNotApplied`).

File: sdk/core/Azure.Core/CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+- Added `HttpPipelineTransportOptions` type.  This type contains a `ServerCertificateCustomValidationCallback` property that allows callers to set a `Func<ServerCertificateCustomValidationArgs, bool>` delegate.  If set, the delegate will be called when validating the server side TLS certificate.
```
Comment (christothes) [inline sdk/core/Azure.Core/CHANGELOG.md:11]:
```suggestion
- Added `HttpPipelineTransportOptions` type.  This type contains a `ServerCertificateCustomValidationCallback` property that allows callers to set a `Func<ServerCertificateCustomValidationArgs, bool>` delegate.  If set, the delegate will be called to validate the server side TLS certificate.
```

**Review by christothes — APPROVED** [review]
Looks great. One minor tweak suggestion on the change log

**Pull request merged**